expected/semantic-search/text/similar-condition-v2/options/prefix.out

```diff
@@ -0,0 +1,47 @@
+-- Only test when `PGRN_LANGUAGE_MODEL_TEST` is set.
+\getenv language_model_test PGRN_LANGUAGE_MODEL_TEST
+SELECT NOT :{?language_model_test} AS omit \gset
+\if :omit
+  \quit
+\endif
+CREATE TABLE memos (
+  id integer,
+  content text
+);
+INSERT INTO memos VALUES (1, 'PostgreSQL is a RDBMS.');
+INSERT INTO memos VALUES (2, 'Groonga is fast full text search engine.');
+INSERT INTO memos VALUES (3, 'PGroonga is a PostgreSQL extension that uses Groonga.');
+CREATE INDEX pgrn_index ON memos
+ USING pgroonga (content pgroonga_text_semantic_search_ops_v2)
+ WITH (plugins = 'language_model/knn',
+       model = 'hf:///groonga/multilingual-e5-base-Q4_K_M-GGUF',
+       passage_prefix = 'passage: ',
+       query_prefix = 'query: ');
```

Comment:
Could you also use them in https://github.com/pgroonga/pgroonga/blob/main/sql/semantic-search/text/similar-distance-condition-v2/with-condition.sql and https://github.com/pgroonga/pgroonga/blob/main/sql/semantic-search/text/similar-distance-condition-v2/without-condition.sql ?

We should use them with `groonga/multilingual-e5-base-Q4_K_M-GGUF`.